.gitignore

```diff
@@ -2,4 +2,6 @@
 node_modules
 package-lock.json
 
-dist
\ No newline at end of file
+dist
+# Local Netlify folder
+.netlify
```

Comment:
I notice there's a `yarn-error.log` in this PR. Let's ignore that file too

Reply:
Fixed!